candi/bashible/common-steps/node-group/064_configure_kubelet.sh.tpl

```diff
@@ -300,10 +300,10 @@ RotateKubeletServerCertificate default is true, but CIS becnhmark wants it to be
 https://kubernetes.io/docs/reference/command-line-tools-reference/feature-gates/
 */}}
 featureGates:
-{{- if semverCompare "<1.27" .kubernetesVersion }}
+{{- if semverCompare "=1.27" .kubernetesVersion }}
   ExpandCSIVolumes: true
```

Comment:
since version 1.27 it is no longer a feature gate

Reply:
fix